torch/testing/_internal/common_optimizers.py

```diff
@@ -1378,6 +1378,11 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
         supported_impls=("foreach", "differentiable", "fused"),
         supports_sparse_on=("cpu", "cuda"),
         skips=(
+            DecorateInfo(
+                unittest.skip("SGD currently doesn't have capturable"),
```

Comment:
There is a skip in the test for it so there shouldn't be one here?

Reply:
good catch, i had forgotten to save the file locally 🤦🏻‍♀️